packages/web-components/src/components/va-pagination/va-pagination.tsx

```diff
@@ -70,6 +72,10 @@ export class VaPagination {
 
   private handlePageSelect = (page, eventID) => {
     this.pageSelect.emit({ page });
+
+    // Reset focus to the active page button.
+    (this.el.shadowRoot.activeElement as HTMLElement).blur();
```

Comment:
This fixes a bug where the selected page was not losing its active style making it appears like two pages were selected at the same time.  I don't think there is a specific issue ticket for this but it was mentioned as part of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1408

Reply:
I'm so glad you caught this! I thought I just had a super strange bug on my side because I couldn't replicate it.